### DISCUSSION TITLE
Divisão de trabalho para equipe

### DISCUSSION THREAD

**Discussion by jdgabriel** [backend-br/forum#137] [Q&A] [answered]
Fala devs, tudo beleza?
Eu sempre fui frella, sendo assim, sempre fiz tudo sozinho na questão de backend. Mas agora, estou trabalhando em uma empresa que recém criou sua área de desenvolvimento de produtos.
O time de desenvolvimento está sendo composto por 3 a 4 Devs. Minha dúvida é... 
Como começar o projeto do zero?
Como organizar o fluxo de trabalho em um projeto? 
Como faço para poder trabalhar simultaneamente e coordenado com o time?

Vlw.

**Comment by igorlvicente** [comment, marked as answer]
Posso estar errado, mas acho que isso é uma grande área que não pode ser resumida tão facilmente.
O que eu sigo à risca é "Comunicar demais é melhor que de menos". Sempre diga o que você está fazendo, o que você fez e o que você pretende fazer no futuro. Pergunte. Se algo não ficar 100% entendido, pergunte de novo. Se for mexer em algo que outro alguém já mexeu, sinalize de alguma forma. Peça _insights_ ou dicas. Mesmo que 95% do que falarem você já saiba, os 5% ajudarão a melhorar o "pensamento compartilhado".

**Comment by jeancsanchez** [comment]
Acho que pode começar pelo básico que funciona: Kanban + Git flow
